docs/docs/cmd/spo/page/page-section-add.mdx

```diff
@@ -26,10 +26,46 @@ m365 spo page section add [options]
 : Order of the section to add.
 
 `--zoneEmphasis [zoneEmphasis]`
-: Section background shading. Allowed values `None`, `Neutral`, `Soft`, `Strong`
+: Section background shading. Allowed values `None`, `Neutral`, `Soft`, `Strong`, `Image`,`Gradient`
 
 `--isLayoutReflowOnTop`
 : The position of the Vertical section for smaller screens. Applied only for Vertical section.
+
+`--isCollapsibleSection`
+: Set section to be collapsible.
+
+`--showDivider`
+: Show divider line between sections.
+
+`--iconAlignment [iconAlignment]`
+: Expand/collapse icon alignment.
+
+`--isExpanded`
+: Set default display of collapsible section.
+
+`--gradientText [gradientText]`
+: Set gradient setting of a background of a section.
+
+`--imageUrl [imageUrl]`
+: Background image URL.
+
+`--imageHeight [imageHeight]`
+: Background image height.
+
+`--imageWidth [imageWidth]`
+: Background image width.
+
+`--fillMode [fillMode]`
+: Fill mode of an image. Possible values are `ScaleToFill`, `ScaleToFit`, `Tile`, `OriginalSize`.
+
+`--useLightText`
+: Use light text parameter for background.
+
+`--overlayColor [overlayColor]`
+:	Overlay color for background. In #RRGGBB format.
+
+`--overlayOpacity [overlayOpacity]`	
+: Overlay opacity for background.
 ```
```

Comment:
Update the description here that these are only required when zoneEmphasis is `Image` or `Gradient`

Reply:
the same here, I mentioned that these properties are applied only for Image and Gradient backgrounds, but if not specified then the default values are set